pkg/geo/geomfn/topology_operations.go

```diff
@@ -96,6 +98,9 @@ func Difference(a, b geo.Geometry) (geo.Geometry, error) {
 
 // Simplify returns a simplified Geometry.
 func Simplify(g geo.Geometry, tolerance float64) (geo.Geometry, error) {
+	if math.IsNaN(tolerance) {
```

Comment:
can you change this to `if math.IsNaN(tolerance) g.ShapeType2D() == geopb.ShapeType_POINT || g.ShapeType2D() == geopb.ShapeType_MULTIPOINT `, and add tests for points and multipoints?

Reply:
I have made the changes. please review this again.

Thanks🙇🏻‍♂️